### DISCUSSION TITLE
How to disable the /server and /hub command.

### DISCUSSION THREAD

**Discussion by fidzhikovsky** [Zrips/CMI#5186] [Q&A]
How to disable the /server and /hub command. It conflicts with other bungee server commands

![image](https://user-images.githubusercontent.com/67680853/109385384-76821a00-7904-11eb-9604-ae20ac47b69b.png)

**Comment by mrfloris** [comment]
If `/server` is being used by CMI (`/cmi server`) because it's enabled in `Alias.yml` you can turn it off by setting it to false in `Alias`.yml`.
```yml
  # /cmi server $1-
  /server:
    Enabled: false
    Tab: true
```
I don't think `/hub` is from CMI ~ I cannot find it in `Alias.yml` and with checkcommand. 
```
> cmi hub
[09:58:04 INFO]: There is no command by this name!
> cmi checkcommand hub
[09:58:10 INFO]: No commands found by this keyword
>
```
